srcpkgs/libunibreak/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'libunibreak'
 pkgname=libunibreak
-version=4.1
+version=4.2
 revision=1
 build_style=gnu-configure
 short_desc="An implementation of the Unicode line breaking algorithm"
 maintainer="Diogo Leal <diogo@diogoleal.com>"
 license="zlib-acknowledgement"
 homepage=https://fbreader.org/content/fbreader-beta-linux-desktop""
```

Comment:
please fix the quotes

Reply:
Shouldn't homepage be http://vimgadgets.sourceforge.net/libunibreak/ ?